### DISCUSSION TITLE
Variable to include site tagline in a slot?

### DISCUSSION THREAD

**Discussion by iainhallam** [combostrap/combo#67] [Q&A]
I've used the $lead variable to include the DokuWiki configuration for tagline in the text of my home page, and it works fine, but then I used it in a footer slot, and it's empty while showing the home page. Going to another page on the site, of course it takes the lead from that page. Is there a variable that will give me the site's tagline in the footer slot from any page in the site?

**Comment by gerardnico** [comment]
There is no variable because the tagline may have many variations. 

Most people write the variations short / long that is adapted to their layout.
